lib/wanda/execution/fact.ex

```diff
@@ -3,6 +3,7 @@ defmodule Wanda.Execution.Fact do
   A fact is a piece of information that was gathered from a target.
   """
 
+  @derive Jason.Encoder
```

Comment:
`FactError` should have the json encoding as well

Reply:
Thanks, done here db14ef366491aaca830f26850fcd35e5bfb289cc